### ISSUE TITLE
Class 'phpFastCache\CacheManager' not found

### ISSUE THREAD

**Issue by mzf** [PHPSocialNetwork/phpfastcache#597]
### Configuration:

PhpFastCache version: 7.0.0-rc
PHP version:7.2
Operating system: Win10

#### Issue description:
Composer error
Directory in composer\autoload_static.php is 'src', but name of dir is 'lib'

**Comment by Geolim4** [comment]
Hello,

Just run this command with composer:
https://getcomposer.org/doc/03-cli.md#dump-autoload-dumpautoload-

Your composer static cache is just outdated.

Cheers,
Georges

**Comment by Geolim4** [comment]
Also please don't use `phpFastCache\CacheManager` but `Phpfastcache\CacheManager`.
This has been changed as of the V7.

**Comment by eesteche** [comment]
Hi! I made a small project with faker and PhpFastCache and it works perfectly.

Yet when i try to implement it to my project i get the same error "Class 'phpFastCache\CacheManager' not found".

The error: "Fatal error: Uncaught Error: Class 'Phpfastcache\CacheManager' not found in C:\xampp\htdocs\codoacodo_php\cache.php:5 Stack trace: #0 C:\xampp\htdocs\codoacodo_php\index.php(4): require_once() #1 {main} thrown in C:\xampp\htdocs\codoacodo_php\cache.php on line 5"

cache.php :
<?php
use Phpfastcache\CacheManager;
use Phpfastcache\Config\ConfigurationOption;

CacheManager::setDefaultConfig(new ConfigurationOption([
    'path' => 'C:/tmp/'
]));

index.php :
<?php
session_start();
require_once("config.php");
require_once("cache.php");
require_once('Articulos.php');
?>

Any idea on what could it be? I followed the same exact process as i did for the faker project. CD into the folder, composer require phpfastcache/phpfastcache
and then added the lines above. Didnt even got to implement the solution. 

PHP: 7.4
phpFastCache v7.1.1 
last version of xampp
